.eslintrc.js

```diff
@@ -0,0 +1,225 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "commonjs": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "block-scoped-var": "error",
+        "block-spacing": [
+            "error",
+            "always"
+        ],
+        "brace-style": [
+            "error",
+            "stroustrup"
+        ],
+        "callback-return": "error",
+        "camelcase": "error",
+        "comma-spacing": [
```

Comment:
@vitkarpov Yup, you’re right, found it.

Nit: could we put `"comma-dangle"` before `"comma-spacing"` so that they are alphabetically sorted like other properties?